DESCRIPTION

```diff
@@ -0,0 +1,47 @@
+Package: admiralonco
+Type: Package
+Title: Oncology extension package for ADaM in R Asset Library 
+Version: 0.0.0
+Authors@R: c(
+    person("Stefan", "Bundfuss", email = "stefan.bundfuss@roche.com", role = c("aut", "cre")),
+    person("Ross", "Farrugia", role = "aut"),
+    person("Amit", "Jain", role = "aut"),
+    person("Liming (Catherine)", "Jin", role = "aut"),
+    person("F. Hoffmann-La Roche AG", role = c("cph", "fnd")),
+    person("GlaxoSmithKline LLC", role = c("cph", "fnd")),
+    person("Amgen Inc.", role = c("cph", "fnd"))
+    )
+Description: A toolbox for programming oncology specific Clinical Data Standards Interchange Consortium
+    (CDISC) compliant Analysis Data Model (ADaM) datasets in R. ADaM datasets are a
+    mandatory part of any New Drug or Biologics License Application submitted to the
+    United States Food and Drug Administration (FDA). Analysis derivations are
+    implemented in accordance with the "Analysis Data Model Implementation Guide"
+    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam/adamig-v1-3-release-package>).
+License: Apache License (>= 2)
+Encoding: UTF-8
+LazyData: true
+Roxygen: list(markdown = TRUE)
+RoxygenNote: 7.1.2
+Depends: R (>= 3.5)
+Imports:
+    admiral,
```

Comment:
add admiral.test for RS/TU/TR test data and could consider adding all the others admiral uses as chances are we'll need similar? or maybe add them as needed over time if you prefer

Reply:
I have added admiral.test. I would add others when needed. That's easier than checking later which ones are not needed.